react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  let text = '';
+  let checkOrX = '';
+
+  if (password.length >= 0 && password.length < 8) {
+    text = 'Your password is too short.';
+    checkOrX = '❌';
+  }
+  if (password.length === 0) {
+    text = 'A password is required.';
+    checkOrX = '❌';
+  } else if (password.length > 7) {
+    checkOrX = '✅';
+  }
+
+  return (
+    <>
+      <div>
+        <form>
+          <div>
+            <label>Password</label>
```

Comment:
Make sure your `<label>`s are attached to their form controls, either by wrapping them or using the `for` attribute.

Reply:
fixed